docs/guides/testing.md

```diff
@@ -34,13 +36,16 @@ beforeEach(() => {
 
 The way you mock a dependency depends on your test runner/library.
 
-In [jest](https://jestjs.io/), you can create a `__mocks__/zustand.js` and place the code in that file. If your app is using `zustand/vanilla` instead of `zustand`, then you'll have to place the above code in `__mocks__/zustand/vanilla.js`.
+In [jest](https://jestjs.io/), you can create a `__mocks__/zustand.js` and place the code in that
+file. If your app is using `zustand/vanilla` instead of `zustand`, then you'll have to place the
+above code in `__mocks__/zustand/vanilla.js`.
```

Comment:
Nitpick: I find it easier to read and edit if the newlines are inserted after logical pauses in sentences, like commas or full stops.

See <https://sembr.org/>, it's a great read :)